### PR TITLE
Initial VS Project setup

### PR DESCRIPTION
**Pull request by nullorvoid**
# Description

The initial project commit, no code except a Hello World test which can be removed when we write the Unit tests.
# What Changed
- Updated gitignore to properly remove folders in subfolders
- Added Tests project
- Moved sln to top level
- Updated readme

### CODE REVIEW

**Comment by kefniark** [comment]
let me merge and edit your description

File: Tests/TestStartUp.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace Tests
+{
+	[TestFixture]
+	public class TestStartUp
+	{
+		[Test]
+		public void HelloTest()
+		{
+			Assert.IsEmpty("");
```
Comment (kefniark) [inline Tests/TestStartUp.cs:11]: Best NUnit ever :P

**Comment by nullorvoid** [comment]
Given rights :+1:

File: Tests/TestStartUp.cs (hunk 1 above)
Comment (nullorvoid) [inline Tests/TestStartUp.cs:11]: Yeap! I needed something to test

**Comment by kefniark** [comment]
[LGTM](http://lgtm.in/i/ihYpZigon)

**Pull request merged**